src/rust/ensogl/example/src/easing_animator.rs

```diff
@@ -2,14 +2,14 @@
 
 use crate::prelude::*;
 
-use ensogl::animation::easing::*;
-use ensogl::animation;
-use ensogl::system::web::AttributeSetter;
-use ensogl::system::web::create_element;
-use ensogl::system::web::get_element_by_id;
-use ensogl::system::web::NodeInserter;
-use ensogl::system::web::StyleSetter;
-use ensogl::system::web;
+use ensogl_core::animation::easing::*;
+use ensogl_core::animation;
+use ensogl_core::system::web::AttributeSetter;
+use ensogl_core::system::web::create_element;
+use ensogl_core::system::web::get_element_by_id;
+use ensogl_core::system::web::NodeInserter;
+use ensogl_core::system::web::StyleSetter;
+use ensogl_core::system::web;
```

Comment:
I wonder if the examples should not just use the main ensogl library? it should compile very fast (as it contains reimports only).

Reply:
@farmaazon It's not strictly true. Imagine I'm changing display object definition - then it will recompile text etc - now itsm uch faster